### PR TITLE
Allow extra headers to be added to requests

### PR DESCRIPTION
**Pull request by vaizki**
I needed to add `apns-expiration` and `apns-collapse-id` headers to the requests.. Here is a way to insert extra headers.

### CODE REVIEW

**Comment by vaizki** [comment]
Ok that was an embarrassing push, sorry.

**Comment by etataurov** [comment]
Thanks for the PR.
Unfortunately the project was a bit unmaintained and CI now fails even in develop branch.
I'm going to fix that first and then have a look at your changes

File: asyncio_apns/apns_connection.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def disconnect(self):
         self.protocol = None
 
     def _prepare_request(self, payload: Union[Payload, str], token: str,
-                         priority: NotificationPriority, topic: str):
+                         priority: NotificationPriority, topic: str, extra_headers: Sequence[Tuple[str, str]]=[]):
```
Comment (etataurov) [inline asyncio_apns/apns_connection.py]: Please don't use mutable object as default argument value https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

File: asyncio_apns/apns_connection.py [hunk 2]
```diff
@@ -82,15 +82,18 @@ def _prepare_request(self, payload: Union[Payload, str], token: str,
         ]
         if topic:
             request_headers.append(('apns-topic', topic))
+        if extra_headers:
+            for h_tuple in extra_headers:
+                request_headers.append(h_tuple)
```
Comment (etataurov) [inline asyncio_apns/apns_connection.py]: `request_headers.extend(extra_headers)` does the same in single line :)

**Review by etataurov — CHANGES_REQUESTED** [review]

**Comment by etataurov** [comment]
`develop` branch is now stable, you can rebase

File: asyncio_apns/apns_connection.py (hunk 2 above)
Comment (vaizki) [reply asyncio_apns/apns_connection.py]: Quite so!

File: asyncio_apns/apns_connection.py (hunk 1 above)
Comment (vaizki) [reply asyncio_apns/apns_connection.py]: I don't know what happened here... thanks.

**Pull request merged**